docs/io.cozy.todos.md

```diff
@@ -0,0 +1,123 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Todos doctype
+
+## `io.cozy.todos.list`
+
+This doctype represents a list of todos:
+
+- `title`: {string} - The list title
+- `description`: {string} - The list description
+- `limitDate`: {date} - The limit date to complete all the items.
+
+### Relationships
+
+- `items`: {object} - The list items
+
+
+### Example
+
+```json
+{
+  "_id": "2165d9a310deadbeeffc08d54c45102",
+  "title": "Create a loan file",
+  "description": "Loan money to buy a house.",
+  "limitDate": "2020-02-01T23:59:59Z",
+  "relationships": {
+    "items": {
+      "data": [
+        {
+          "_id": "8bffbc24e67716b63e08c71bd0059fea",
+          "_type": "io.cozy.todos.item"
+        },
+        {
+          "_id": "68a74e03722f02b1a6fc6a4508015de0",
+          "_type": "io.cozy.todos.item"
+        }
+      ]
+    }
+  }
+}
+```
+
+
+## `io.cozy.todos.item`
+
+This doctype represents a todo item:
+
+- `label`: {string} - The label of the todo item, e.g. "Do stuff".
+- `done`: {bool} - Whether the item is done or not.
+- `limitDate`: {date} - The limit date to complete this item.
+- `completionDate`: {date} - The actual completion date of the item.
+
+### Relationships
+
+- `files`: {object} - Files linked to the todo item.
+- `contacts`: {object} - Contacts linked to the todo item.
+- `rules`: {object} - The rules used to match others relationships. See below or in the `rules` [doctype](https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.rules) for more details.
+
+
+#### Rules
+
+A file or contact relationship may be associated to a rule, used to matched the documents, e.g. a rule targeting the last 3 files payslips. In which case, the associated rule is specified in a `data.metadata` object in the relationship:
```

Comment:
I'd find it clearer if the example was not mixed with the definition here: "When a document has been linked to a todo through the use of a *rule*, the *rule* that was used to find the document is saved in the `metadata` of the relationship."

Reply:
Much clearer indeed :+1: